src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -1256,6 +1256,13 @@ object FunctionRegistry {
     "min" -> "Minimum value of interval.",
     "max" -> "Maximum value of interval, non-inclusive.")
 
+  register("dbeta", { (x: Double, a: Double, b: Double) => dbeta(x,a,b)},
+  """
+  Returns a random draw from a beta distribution on [0,1] defined by the parameters a and b.
+  """,
+  "x" -> "Point at which to sample ",
```

Comment:
Point in [0, 1] at which to evaluate the density. If a < 1 then x must be positive. If b < 1 then x must be less than 1.